package.json

```diff
@@ -26,7 +26,7 @@
     ]
   },
   "scripts": {
-    "build": "turbo run build",
+    "build": "turbo run build --filter=!polaris.shopify.com && turbo run build --filter=polaris.shopify.com",
```

Comment:
Important to ensure the packages are built before trying to build the site (which depends on them)